apps/utils/scripts/run-bfd-server

```diff
@@ -248,9 +258,16 @@ setvar paths/files/truststore "$bin_dir/ssl/${truststore_file}"
 setvar paths/files/keystore "$bin_dir/ssl/${keystore_file}"
 setvar paths/files/war "$bin_dir/$war_file_name"
 
-setvar db/url "jdbc:postgresql://${db_host}:${db_port}/bfd?logServerErrorDetail=false"
-setvar db/username "bfd"
-setvar db/password "InsecureLocalDev"
+# Conditional database URL setting
+if $host_set && [[ -n "$username" ]] && [[ -n "$password" ]]; then
+  setvar db/url "jdbc:postgresql://${db_host}:${db_port}/fhirdb?logServerErrorDetail=false"
+    setvar db/username $username
+    setvar db/password $password
+else
+ setvar db/url "jdbc:postgresql://${db_host}:${db_port}/bfd?logServerErrorDetail=false"
+   setvar db/username "bfd"
```

Comment:
If we add the default as I mentioned above then I think we can move these out of the conditional

Reply:
the only reason I didn't set the values as default is because it made sense to check for null incase a user only set the host but not the username and password

Reply:
Yeah, I'm wondering if we can just fix it so that the database name doesn't have to be different for local development.

Reply:
Actually I think we can. We just need to change the database name in `run-db-migrator` and then we can always connect to `fhirdb` here instead of `bfd`. And then we shouldn't need to do any additional checks.

Reply:
okay, looking into this

Reply:
@aschey-forpeople @MahiFentaye That's a little bit out of scope for this ticket. Let's create a new one, as I think it will be a little bit more work than just changing one line.

Reply:
Actually I think it is one line if I'm not missing something. The migrator script hardcodes the db name, we just need to change it there.